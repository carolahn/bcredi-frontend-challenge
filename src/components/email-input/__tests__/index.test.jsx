import React from "react";
import renderer from "react-test-renderer";

import EmailInput from "../index";

describe("EmailInput render test", () => {
  it("renders correctly EmailInput", () => {
    const tree = renderer
      .create(<EmailInput values={{}} register={() => {}} errors={{}} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
