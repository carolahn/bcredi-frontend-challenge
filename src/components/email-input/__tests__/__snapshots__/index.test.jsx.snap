// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailInput render test renders correctly EmailInput 1`] = `
<div
  className="register__input-wrapper"
>
  <label
    className="register__label"
  >
    E-mail
  </label>
  <input
    className="sc-gsTCUz fKZdNo register__input"
    name="email"
    placeholder="exemplo@email.com"
    style={
      Object {
        "border": "1px solid #e6e6e6",
        "color": "#333333",
      }
    }
    width="320px"
  />
</div>
`;
